com.gitlab.cunidev.Workflow.json

```diff
@@ -0,0 +1,48 @@
+{
+    "app-id" : "com.gitlab.cunidev.Workflow",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.32",
```

Comment:
Have you tried using 3.34?

Reply:
Just restart Builder and it should work.

Reply:
Thanks. I tested it and it seems to work just as expected on 3.34